test/unifile-fs.js

```diff
@@ -367,7 +367,7 @@ describe('FsConnector', function() {
 		it('returns the content of a file', function() {
 			return connector.readFile({}, '/tmp/test')
 			.then((content) => {
-				expect(content).to.equal(data);
+				expect(content.toString()).to.equal(data);
```

Comment:
Could you add `expect(content).to.be.an.instanceof(Buffer)` please?

Reply:
sure